packages/jest-environment-puppeteer/src/readConfig.js

```diff
@@ -46,24 +45,12 @@ export async function readConfig() {
 
   // eslint-disable-next-line global-require, import/no-dynamic-require
   const localConfig = await require(absConfigPath)
-  return merge({}, defaultConfig, localConfig)
-}
 
-export function getPuppeteer(config) {
-  switch (config.browser.toLowerCase()) {
-    /* eslint-disable global-require, import/no-dynamic-require, import/no-extraneous-dependencies, import/no-unresolved */
-    case 'chromium':
-      try {
-        return require('puppeteer')
-      } catch (e) {
-        return require('puppeteer-core')
-      }
```

Comment:
I think we need to keep the `getPuppeteer` function to allow using `puppeteer` or `puppeteer-core`